docs/content/reference/resource-schema/core-schema/container-schema/index.md

```diff
@@ -35,7 +35,7 @@ weight: 300
 | Key  | Required | Description | Example |
 |------|:--------:|-------------|---------|
 | image | y | The registry and image to download and run in your container. Follows the format `<registry-hostname>:<port>/<image-name>:<tag>` where registry hostname is optional and defaults to the Docker public registry, port is optional and defaults to 443, tag is optional and defaults to `latest`.| `ghcr.io/USERNAME/myimage:latest`
-| env | n | A list of environment variables to be set for the container. | `'ENV_VAR': 'value'`
+| env | n | A list of environment variables to be set for the container. Environment variables can either be of `value` or a reference so a kubernetes secret in the format `valueFrom`. | `'ENV_VAR': { value: 'value' }` or `'ENV_VAR': { valueFrom: { secretRef: { source: secret.id key: 'SECRET_KEY' } } }`
```

Comment:
Should we make the reference description more higher level, since kubernetes secret is implementation detail and will change when support for types is added?

```suggestion
| env | n | A list of environment variables to be set for the container. Environment variables can either be of `value` or a reference to a Application.Core/SecretStore resource id in the format `valueFrom`. | `'ENV_VAR': { value: 'value' }` or `'ENV_VAR': { valueFrom: { secretRef: { source: secret.id key: 'SECRET_KEY' } } }`
```